spring-boot-docs/src/main/asciidoc/spring-boot-features.adoc

```diff
@@ -2185,17 +2184,15 @@ to the home page at "/" and keep the default for everything else:
 
 [source,java,indent=0]
 ----
-	@Configuration
+	@EnableWebSecurity
```

Comment:
This will switch off all of Boot's web security configuration as `SpringBootWebSecurityConfiguration` is `@ConditionalOnMissingBean(WebSecurityConfiguration.class)`. Doesn't that mean that "and keep the default for everything else" is no longer true?